doc/versionHistory.md

```diff
@@ -1,5 +1,10 @@
 # Version History
 
+0.2.1
+
+- Create sendEvt.vi to write the event to TCP Client.
```

Comment:
Please consider to use **sendEvt.vi** or `sendEvt.vi` to express the vi.

Reply:
Done.